util/util.go

```diff
@@ -9,3 +9,11 @@ func AddrToPath(s string) string {
 	s = strings.Replace(s, ":", "_", -1)
 	return strings.Replace(s, "/", "", -1)
 }
+
+// some.host:2003, kafkaRoute -> kafkaRoute_some_host_2003
+// http://some.host:8080, kafkaRoute -> kafkaRoute_http_some_host_8080
+func SpoolKeyAddrToPath(s string, routeName string) string {
+	s = AddrToPath(s)
+	sList := []string{routeName, s}
+	return strings.Join(sList, "_")
```

Comment:
this is unnecessarily complicated
can just `return routeName + "_" + AddrToPath(s)`

Reply:
agreed